doc/develop/microscope.rst

```diff
@@ -43,68 +43,122 @@ The main convention is use the role of each component to indicate the function
 of each component in the microscope.
 
 The microscope component can have as role:
+------------------------------------------
  * optical: for an optical microscope (only)
  * sem: an SEM (only)
  * secom
  * sparc
  * sparc-simplex: a SPARC without any alignment controls
- * sparc2
+ * sparc2 (sparcv2)
  * delphi
 
-Typical detectors found in a microscope can be of the following roles:
- * ccd: the main optical camera.
- * se-detector: secondary electron detector of the SEM
+Roles of components found in the DELPHI/SECOM system:
+----------------------------------------------------------------------------------
+# TODO: more detailed explanations
+Emitters:
+ * e-beam: Electron beam of the SEM to scan the sample (emitter).
+ * light: controls the excitation light of the fluorescence microscope.
+ * laser-mirror: scanner of a confocal microscope
+
+Actuators:
+ * stage: It can have 3 linear axes (x, y, z) and 3 rotational axes (rx, ry, rz). # TODO: stage is where?
+ * ebeam-focus: Changes the focus position of the e-beam. It has one axis: z.
+ * chamber: Manages the pressure and/or sample loading. It must have a "pressure" axis.
+ * pinhole: To change the size of the pinhole in a confocal microscope. It has one axis: d.
+ * sem-stage: the stage of the DELPHI that moves the (whole) sample holder.
+ * align: alignment actuator for the SECOM and DELPHI microscopes.
+   For the SECOM, it must have two axes: "a" and "b".
+   For the DELPHI, it must have two axes: "x" and "y".
+
+Detectors:
+ * se-detector: Secondary electron detector of the SEM (detector).
  * bs-detector: backscattered electron detector of the SEM
  * ebic-detector: EBIC detector of the SEM
- * cl-detector: a cathodoluminescence detector, synchronised with the e-beam
+ * ccd: the main optical pixelated detector (e.g. ccd/cmos/spectral camera).
  * photo-detector(N): a 0D photon detector (eg, PMT or APD). It's currently used
    only in confocal microscopes.
- * spectrometer: A detector to acquire multiple wavelengths information
-   simultaneously. It provides the same interface as a DigitalCamera,
+
+System:
+ * lens: Contains parameters concerning the parabolic mirror and the lens system.
+
+Roles of components found in the SPARC/SPARCv2 system:
+-----------------------------------------------------------------------------------
+Emitters:
+ * e-beam: Electron beam of the SEM to scan the sample (emitter).
+
+Actuators:
+ * stage: It can have 3 linear axes (x, y, z) and 3 rotational axes (rx, ry, rz). # TODO: stage is where?
+ * ebeam-focus: Changes the focus position of the e-beam. It has one axis: z.
+ * mirror: To engage the parabolic mirror into the beam path.
+   It has two axes in the SPARCv2 system: s (short), l (long).
+ * mirror-xy: To perform the fine adjustments of the position of the parabolic mirror.
```

Comment:
Add that "mirror-xy" is just "mirror" with a different referential, where x and y are aligned with the x/y of the sample (and of the e-beam).